src/privileged/blurts/subscripts/PanelUI.jsm

```diff
@@ -0,0 +1,138 @@
+class PanelUI {
+  constructor(doc) {
+    this.site = null;
+    this.doc = doc;
+    const box = doc.createElementNS(XUL_NS, "vbox");
+    box.className = "container";
+
+    let elt, elt2, elt3;
+
+    elt = doc.createElementNS(XUL_NS, "description");
+    elt.appendChild(doc.createTextNode("Have an account? It may be at risk."));
```

Comment:
Out of scope for this PR, but I filed #91 to discuss future l10n plans.
I think most of these will be easy to localize, but the concatenation below on L61-74 will be tricky.